packages/components/src/components/relative-datepicker/relative-datepicker.scss

```diff
@@ -0,0 +1,18 @@
+#dc-relative-datepicker {
```

Comment:
Please use class for referencing styles.

Reply:
done :+1: